RecoEgamma/EgammaPhotonProducers/src/PhotonCoreProducer.cc

```diff
@@ -74,6 +75,7 @@ PhotonCoreProducer::PhotonCoreProducer(const edm::ParameterSet& config)
   pixelSeedProducer_ = consumes<reco::ElectronSeedCollection>(conf_.getParameter<edm::InputTag>("pixelSeedProducer"));
   minSCEt_ = conf_.getParameter<double>("minSCEt");
   risolveAmbiguity_ = conf_.getParameter<bool>("risolveConversionAmbiguity");
+  hgcalOnly_ = conf_.getParameter<bool>("hgcalOnly");
```

Comment:
there is nothing else in this module that suggests that HGCAL is used.
The separation is into Barrel and Endcap.
```suggestion
  endcapOnly_ = conf_.getParameter<bool>("endcapOnly");
```

Reply:
@slava77 you are right. The change is now implemented.